cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -730,7 +726,7 @@ construct_runtime!(
 
 		// Message Queue. Importantly, is registered last so that messages are processed after
 		// the `on_initialize` hooks of bridging pallets.
-		MessageQueue: pallet_message_queue = 250,
+		MessageQueue: pallet_message_queue = 175,
```

Comment:
@paritytech/frame-coders any concern about swapping this index on live Rococo?

Reply:
Seems that it is still the last pallet. So, shouldn't be a concern @acatangiu.